accumulator-rsa/Cargo.toml

```diff
@@ -0,0 +1,39 @@
+[package]
+authors = ["Michael Lodder <redmike7@gmail.com>"]
+categories = ["cryptography", "rsa"]
+description = "A Universal Cryptographic RSA Accumulator"
+documentation = "https://docs.rs/accumulator-rs"
+edition = "2018"
+license = "Apache-2.0"
+name = "rsa"
+readme = "README.md"
+repository = "https://github.com/spruceid/ssi"
+version = "0.1.0"
+
+[badges]
+maintenance = { status = "experimental" }
+
+[features]
+default = ["bi-ossl"]
```

Comment:
```suggestion
default = ["bi-rust"]
```
Unify with default in `accumulator-common/Cargo.toml`, since these features conflict. This should fix the build error seen in https://github.com/spruceid/ssi/runs/3633528841:
```
error[E0252]: the name `BigInteger` is defined multiple times
  --> accumulator-common/src/bigint/mod.rs:27:9
   |
23 | pub use ossl::OsslBigInt as BigInteger;
   |         ------------------------------ previous import of the type `BigInteger` here
...
27 | pub use rust::RustBigInt as BigInteger;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ `BigInteger` reimported here
   |
   = note: `BigInteger` must be defined only once in the type namespace of this module
help: you can use `as` to change the binding name of the import
   |
27 | pub use rust::RustBigInt as OtherBigInteger;
   |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```